internal/configs/transportserver.go

```diff
@@ -172,13 +172,15 @@ func generateStreamUpstream(upstream *conf_v1alpha1.Upstream, upstreamNamer *ups
 
 	name := upstreamNamer.GetNameForUpstream(upstream.Name)
 	maxFails := generateIntFromPointer(upstream.MaxFails, 1)
+	maxConnections := generateIntFromPointer(upstream.MaxConnections, 0)
```

Comment:
@pleshakov Is there a default for max_conn in http://nginx.org/en/docs/stream/ngx_stream_upstream_module.html similar to `fail_timeout` ?

Reply:
The default is 0 according to http://nginx.org/en/docs/stream/ngx_stream_upstream_module.html
Which means there's no limit

Reply:
perfect, my bad for not reading the text :grimacing: